assets/wire/utils.js

```diff
@@ -49,6 +49,35 @@ function getRelatedItemsByType(item, type) {
     return item.type === type ? [item] : Object.values(get(item, 'associations', {}) || {}).filter((assoc) => get(assoc, 'type') === type);
 }
 
+export function getFeatureMedia(item) {
+    if (['picture', 'video', 'audio'].includes(item.type)) {
+        return item;
+    }
+
+    const featured = get(item, 'associations.featuremedia');
```

Comment:
it'd be nice to replace lodash get calls with optional chaining so we don't lose typescript support (find usages, rename)

Reply:
This PR is the javascript version for v2.4 of Newshub. The other PR is for typescript (v2.5+) which does not use the `locash.get` function.